airbyte-cdk/python/docs/concepts/schemas.md

```diff
@@ -0,0 +1,23 @@
+# Defining your stream schemas
```

Comment:
@avaidyanatha do you remember why we decided to duplicate these directories instead of linking to the docs outside of the `docs/` directory?

Reply:
I'm not sure I follow. What directories are we duplicating?

Reply:
`airbyte-cdk/python/docs/concepts/` and `docs/contributing-to-airbyte/python/concepts/schemas.md`

Reply:
Aaaahhhh I remember now. Gitbook automatically replicated those files into `/docs`. I initially wanted to keep them in the code directory but that made Gitbook mad. So now we just do everything in the `/docs` directory.